sdk/core/azure-core/inc/azure/core/url.hpp

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Add  this to the simplified core.hpp and update the test.

Also add a changelog entry for this breaking change.